docs/api/wallets.md

```diff
@@ -1,6 +1,6 @@
 # Wallets
 
-Wallets are addresses containing, or previously having contained ARK. A wallet's public key may be unknown to the network, in that case, it is referred to as a `cold wallet`.
```

Comment:
I don't actually like the second sentence, since an address doesn't even need to have a public key. But I suppose that's taken from the ARK doc and the minor technical point is not so relevant.

Reply:
https://github.com/ArkEcosystem/docs/pull/61